tests/tests/datm_ocn_ice_mx025_cfsr

```diff
@@ -0,0 +1,63 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_1_STEP_COLD test
+#
+
+export TEST_DESCR="DATM-MOM6-CICE-CMEPS - 1-step cold start"
+
+export CNTL_DIR="datm_ocn_ice_mx025_cfsr"
+export CNTLMED_DIR="datm_ocn_ice_mx025_cfsr/RESTART"
```

Comment:
CNTLMED_DIR should be removed in all tests

Reply:
Done.